tools/devtool

```diff
@@ -475,14 +475,22 @@ cmd_build() {
         cargo_args+=(--target "$target")
     fi
 
-    [ $profile = "release" ] && cargo_args+=("--release")
+    machine=`uname -m`
+    # On aarch64 musl depends on some libgcc functions (i.e `__addtf3` and other `*tf3` functions) for logic that uses
+    # long double. Such functions are not builtin in the rust compiler, so we need to get them from libgcc.
+    # No need for the `crt_static` flag as rustc appends it by default.
+    [ $machine = "aarch64" ] && RUSTFLAGS="-C link-arg=-lgcc -C link-arg=-lfdt"
```

Comment:
nit: you can [collapse](https://doc.rust-lang.org/rustc/codegen-options/index.html#link-args) multiple linker args into `link-args=-lgcc -lfdt`